examples/resources/oidc-connector.yaml

```diff
@@ -7,19 +11,21 @@ spec:
   provider: example
 
   # This value will be used in the Web UI for "[login with example]" button
+  # button on the cluster login screen so it will say "Login with Google". 
+  # Teleport will provide custom CSS for 'Google' and 'Auth0'. 
   display: "Login with Example"
```

Comment:
Again, this shouldn't be "Login with Example", just "Example" as we automatically add "Login with" in the web interface.